install_static_swift_modules/after/Pods/Target Support Files/CustomModuleMapPod-library-macOS/CustomModuleMapPod-library-macOS.xcconfig

```diff
@@ -1,7 +1,6 @@
 CONFIGURATION_BUILD_DIR = ${PODS_CONFIGURATION_BUILD_DIR}/CustomModuleMapPod-library-macOS
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1
 HEADER_SEARCH_PATHS = $(inherited) "${PODS_ROOT}/Headers/Private" "${PODS_ROOT}/Headers/Private/CustomModuleMapPod" "${PODS_ROOT}/Headers/Public"
-OTHER_SWIFT_FLAGS = $(inherited) -import-underlying-module -Xcc -fmodule-map-file="${SRCROOT}/${MODULEMAP_FILE}"
```

Comment:
is this correct?

Reply:
yes, we know this pod has no swift in it